pkg/gadgets/biolatency/gadget.go

```diff
@@ -45,7 +47,7 @@ distribution of I/O latency (time), giving this as a histogram when it is
 stopped.`
 }
 
-func (f *TraceFactory) DeleteTrace(name string, t interface{}) {
+func (f *TraceFactory) deleteTrace(name string, t interface{}) {
```

Comment:
As noted by @blanquicet, this could also be a function `func deleteTrace(name string, t interface{}`, but I think we can just keep as it's now.